test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/JwtTokenAuthTests.cs

```diff
@@ -85,7 +85,7 @@ public async Task InvokeAdminApi_InvalidAudience_Fails(string headerName)
             var response = await _fixture.Host.HttpClient.SendAsync(request);
             Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
 
-            var validationError = _fixture.Host.GetScriptHostLogMessages().Single(p => p.Level == LogLevel.Debug);
+            var validationError = _fixture.Host.GetScriptHostLogMessages().Single(p => p.Category == ScriptConstants.LogCategoryHostAuthentication && p.Level == LogLevel.Debug);
```

Comment:
The debug level log messages emitted is different when we have a different version of AspNetCore used (net8 version). This change will ensure we are only validating the debug level messages from our specific use case.